components/Players/PlayerManager.vue

```diff
@@ -239,28 +238,30 @@ export default Vue.extend({
           this.setLastProgressUpdate({ progress: new Date().getTime() });
           break;
         case 'playbackManager/SET_CURRENT_TIME': {
-          const now = new Date().getTime();
+          if (state.playbackManager.status === PlaybackStatus.playing) {
+            const now = new Date().getTime();
 
-          if (
-            this.getCurrentItem !== null &&
-            now - state.playbackManager.lastProgressUpdate > 1000 &&
-            state.playbackManager.currentTime !== null
-          ) {
-            this.$api.playState.reportPlaybackProgress(
-              {
-                playbackProgressInfo: {
-                  ItemId: this.getCurrentItem.Id,
-                  PlaySessionId: state.playbackManager.playSessionId,
-                  IsPaused: false,
-                  PositionTicks: Math.round(
-                    this.msToTicks(state.playbackManager.currentTime * 1000)
-                  )
-                }
-              },
-              { progress: false }
-            );
+            if (
+              this.getCurrentItem !== null &&
+              now - state.playbackManager.lastProgressUpdate > 1000 &&
+              state.playbackManager.currentTime !== null
+            ) {
+              this.$api.playState.reportPlaybackProgress(
+                {
+                  playbackProgressInfo: {
+                    ItemId: this.getCurrentItem.Id,
+                    PlaySessionId: state.playbackManager.playSessionId,
+                    IsPaused: false,
+                    PositionTicks: Math.round(
+                      this.msToTicks(state.playbackManager.currentTime * 1000)
+                    )
+                  }
+                },
+                { progress: false }
+              );
 
-            this.setLastProgressUpdate({ progress: new Date().getTime() });
+              this.setLastProgressUpdate({ progress: new Date().getTime() });
```

Comment:
Can't we use ``now`` here?

Reply:
@MrTimscampi I don't understand why it's outside the scope. You're already declaring ``const now`` in line 242 (as I mentioned above), why not use that instead?

Reply:
This is all untouched code though, it just has an indentation change :)